### DISCUSSION TITLE
I need a bit of help understanding distributed mode! :)

### DISCUSSION THREAD

**Discussion by jreoka** [minio/minio#13400] [Q&A] [answered]
Hello! I'd call myself somewhat decent with understanding Minio but I'm no expert. I'm having a bit of trouble understanding how to run Minio in distributed mode.

Let's say I have 4 servers, does each of the four servers have to have drives connected to it? Or can one be the "main" server that oversees the others and not need to have drives connected to it?

Also, lets say the IP's are like this:

Server 1 = 1.1.1.1

Server 2 = 2.2.2.2

Server 3 = 3.3.3.3

Server 4 = 4.4.4.4

How would I start each server? AKA what woulld I put in each command to start it? (I'm not using domain names because they are all on the same network)

**Comment by klauspost** [comment, marked as answer]
Everything you ask about is in the documentation. Maybe start here: https://docs.min.io/minio/baremetal/installation/deploy-minio-distributed.html#deploy-minio-distributed

The short answers:

> How would I start each server?

If you don't want hostnames, IPs should be consecutive. But in general you should just set host names, even if local.

On bare metal execute minio with the appropriate parameters. Note that parameters are the same for all, except if you need to set "--(console-)address".

**Reply by jreoka** [reply]
Thank you

**Reply by xtrmstep** [reply]
the link is outdated. I think this one is correct now http://docs.min.io/enterprise/aistor-object-store/operations/core-concepts/
